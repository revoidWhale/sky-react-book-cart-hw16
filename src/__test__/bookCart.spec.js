import BookCart from '../components/BookCart'
import Enzyme, { shallow } from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
import React from 'react'

Enzyme.configure({ adapter: new Adapter() })

describe('BookCart component', () => {
  it('should render BookCart component', () => {
    const view = shallow(<BookCart />)
    expect(view).toMatchSnapshot()
  })

  it('should contain <table>', () => {
    const component = shallow(<BookCart />)
    const wrapper = component.find('table')
    expect(wrapper.length).toBe(1)
  })

  it('should contain class .some', () => {
    const component = shallow(<BookCart />)
    const wrapper = component.find('.some')
    expect(wrapper.length).toBe(1)
  })
})
