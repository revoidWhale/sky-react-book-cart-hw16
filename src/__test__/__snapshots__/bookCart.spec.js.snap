// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookCart component should render BookCart component 1`] = `ShallowWrapper {}`;
